Conceptpower+Spring/src/integration-test/java/edu/asu/conceptpower/rest/ConceptSearchIT.java

```diff
@@ -28,6 +28,19 @@ public void test_searchConcept_searchWithWordAndPosInJson() throws Exception {
     }
 
     @Test
+    public void test_searchConcept_successWithDefaultOperator() throws Exception {
+
+        final String output = IOUtil
+                .toString(this.getClass().getClassLoader().getResourceAsStream("output/conceptWithWordAndPos.json"));
+
+        this.mockMvc
+                .perform(MockMvcRequestBuilders.get("/ConceptSearch").param("word", "einstein").param("pos", "noun")
+                        .accept(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(content().string(output)).andExpect(status().isOk());
```

Comment:
is there a word that's not a noun that would come up if you search for Einstein?